nafis-vw-header/includes/css/scss/_module-header-brand-dealership.scss

```
@@ -0,0 +1,81 @@
+.brandDealership {
```

Refactor class name to indicate it's a parent wrapper
```suggestion
.brandDealershipWrapper {
```